cri/v1alpha1/cri_utils_test.go

```diff
@@ -821,3 +824,489 @@ func Test_parseResourcesFromCRI(t *testing.T) {
 		})
 	}
 }
+
+func Test_generateContainerMounts(t *testing.T) {
+	type args struct {
+		sandboxRootDir string
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+	// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := generateContainerMounts(tt.args.sandboxRootDir); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("generateContainerMounts() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_generateMountBindings(t *testing.T) {
+	type args struct {
+		mounts []*runtime.Mount
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+	// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := generateMountBindings(tt.args.mounts); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("generateMountBindings() = %v, want %v", got, tt.want)
```

Comment:
But some drawbacks  exist, I suppose. 
Here, if we try to test the wrong test case, surely we will get an error, as we expect. But we cannot verify the error type is one we expect.
I guess one has to add some extra check code if high accuracy is wanted.

Reply:
It just help us do basic things to avoid unnecessary duplication of effort. For high accuracy, we could do better on this basis :)